webapp/src/routes/IncomeReport/index.js

```diff
@@ -81,70 +86,99 @@ const IncomeReport = () => {
             </Box>
           </Grid>
         </Grid>
-        <Box marginTop={2} paddingLeft={2}>
-          {t('textInformation')}
-          <br />
-          Next disbursement date: June 8, 2022
-        </Box>
-        <Box display="flex" justifyContent="end">
-          <FormControl>
-            <RadioGroup
-              name="controlled-radio-buttons-group"
-              value={showDelegateRadio}
-              row
-              onChange={({ target }) => setShowDelegateRadio(target.value)}
-            >
-              <FormControlLabel
-                control={<Radio size="small" />}
-                label={t('textRadioButton1')}
-                value="oneDelegate"
-              />
-              <FormControlLabel
-                control={<Radio size="small" />}
-                label={t('textRadioButton2')}
-                value="allDelegates"
-              />
-            </RadioGroup>
-          </FormControl>
-          <IncomeSelect
-            onChangeFunction={setElectionYearSelect}
-            labelSelect={t('textYearSelect')}
-            values={getListElectionYears()}
-            actualValue={electionYearSelect}
-          />
-          <IncomeSelect
-            onChangeFunction={setElectionRoundSelect}
-            labelSelect={t('textElectionSelect')}
-            values={electionsByYearList.map(data => `${data.election}`)}
-            actualValue={electionRoundSelect}
-          />
-          <IncomeSelect
-            onChangeFunction={setTypeCurrencySelect}
-            labelSelect={t('textCurrencySelect')}
-            values={['EOS', 'USD']}
-            actualValue={typeCurrencySelect}
-          />
-          <IncomeSelect
-            onChangeFunction={setDelegateSelect}
-            labelSelect={t('textDelegateSelect')}
-            values={incomeByAllDelegatesList.map(
-              data => data.eden_delegate.account
-            )}
-            disable={showDelegateRadio === 'allDelegates'}
-            actualValue={delegateSelect}
-          />
-        </Box>
+
+        <Container>
+          <Box textAlign="justify" marginTop={2}>
+            <Typography variant="span" className={classes.subTitle}>
+              {t('subTitle')}
+            </Typography>
+            <br />
+            <Typography variant="span">
+              {t('textInformation')}
+              <br />
+              Next disbursement date: June 8, 2022
```

Comment:
This date needs to be sourced from the chain tables. we can create a new issue for this